Readme.md

```diff
@@ -106,13 +106,14 @@ fastify.listen(3000, (err) => {
 for a *cache-response-directive* as defined by RFC 2616.
 + `expiresIn` (Default: `undefined`): a value, in seconds, for the *max-age* the
 resource may be cached. When this is set, and `privacy` is not set to `no-cache`,
-then `', max-age=<value>'` will be appended to the `cache-control` header.
+then `', max-age=<value>'` will be appended to the `cache-control` header. (300 seconds = 5 minutes)
```

Comment:
```suggestion
then `', max-age=<value>'` will be appended to the `cache-control` header.
```
Superfluous addition, hopefully readers should know how many seconds are in a minute.

Reply:
Okay, remove. It was originally in there when I had an example of the time for s-maxage, when I wanted to contrast 5 minutes with a month.